ops/services/app_functions/report_stream_batched_publisher/functions/FHIRTestEventReporter/dataHandlers.ts

```diff
@@ -1,34 +1,69 @@
 import { DequeuedMessageItem } from "@azure/storage-queue";
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type SimpleReportTestEvent = Record<any, any>;
-
-export type ProcessedTestEvents = {
-  testEvents: SimpleReportTestEvent[];
+export type FHIRTestEventsBundle = {
```

Comment:
The word `Bundle` in this file is confusing to me because each item in the queue is also caled a `Bundle`. I would suggest replacing `bundle` with something like `batch`, `group`, `list`, `set`, or even `BundleOfBundles`.

Reply:
I came to the same realization and pushed changes to change the wording! Can you refresh the PR page and verify if you still see it?